broker/src/test/java/org/apache/rocketmq/broker/filter/MessageStoreWithFilterTest.java

```diff
@@ -369,4 +367,36 @@ public void testGetMessage_withFilterBitMap() {
             }
         }
     }
+
+    @Test
+    public void testGetMessage_withFilter_checkTagsCode() {
+        try {
+            putMsg(master, topicCount, msgPerTopic);
+            // sleep to wait for consume queue has been constructed.
```

Comment:
Must we use sleep to replace sleep ? Now ,we have polished our unit test within 3 minutes, thoughts @shroman

Reply:
sleep to replace sleep ?